test/e2e/tests/test_table.py

```diff
@@ -155,3 +267,391 @@ def test_enable_ttl(self, forum_table):
+        (ref, res) = billing_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Set billing mode to PROVISIONED
+        cr["spec"] = {
+            "billingMode": "PROVISIONED",
+            "provisionedThroughput": {
+                "readCapacityUnits": 5,
+                "writeCapacityUnits": 5
+            }
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.billing_mode_matcher("PROVISIONED"),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+        # Need more billing mode updates quota (only from PROVISIONED -> PAY_PER_REQUEST)
+        """ # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"] = {
+            "billingMode": "PAY_PER_REQUEST",
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.billing_mode_matcher("PAY_PER_REQUEST"),
+            timeout_seconds=60*3,
+            interval_seconds=5,
+        ) """
+
+    def test_update_provisioned_throughput(self, forum_table):
+        (ref, res) = forum_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Set provisionedThroughput
+        cr["spec"] = {
+            "provisionedThroughput": {
+                "readCapacityUnits": 10,
+                "writeCapacityUnits": 10
+            }
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.provisioned_throughput_matcher(10, 10),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+    def test_enable_sse_specification(self, forum_table):
+        (ref, res) = forum_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Set server side encryption
+        cr["spec"] = {
+            "sseSpecification": {
+                "enabled": True,
+                "sseType": "KMS"
+            }
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.sse_specification_matcher(True, "KMS"),
+            timeout_seconds=120,
+            interval_seconds=3,
+        )
+
+        time.sleep(60)
```

Comment:
Why is this necessary?  Make the value a global param w/ a descriptive name

Reply:
Because of tiny eventual consistency issues of DDB SSE Api